packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -345,6 +345,11 @@
         flex-direction: column;
         position: relative;
         bottom: 0.1rem;
+        &:has(.trade-container__fieldset-info--left) {
+            .trade-container__fieldset-info--left {
+                transform: translateX(7px);
```

Comment:
` trade-container__fieldset-info--left` class moves text for 5 px to the right, but for Turbos and Vanillas it isn't  enough, but 7px is perfect.

Reply:
let's add a single class for accumulators and turbos which will allow us to position the info icon absolutely and to center the text so that we do not rely on pixels (it will behave unexpectedly for different languages) 👍🏻